pagefaultgames/pokerogue
# [Bug] Fix to epic shiny Yungoos icon (#2991)

## What are the changes?
Fixed a floating pixel in epic shiny Yungoos' icon. Caused by Mimikyu's epic shiny sprite being placed too high on the sprite sheet. Moved Mimikyu's sprite down and updated JSON to reflect the change.

## Why am I doing these changes?
Fixes #2291 

## What did change?
Updated the sprite sheet and JSON for gen 7 variants.

### Screenshots/Videos
**Before:**
![image](https://github.com/user-attachments/assets/d588a6c2-05b8-4b57-99f5-f003e1bbf97b)

**After:**
![image](https://github.com/user-attachments/assets/41137aef-2b7f-4e9c-9cce-1dd4e6e1f291)

## How to test the changes?
In override.ts set the overrides to:
`const overrides = {
  STARTER_SPECIES_OVERRIDE: Species.YUNGOOS,
  SHINY_OVERRIDE: true,
  VARIANT_OVERRIDE: 2
} satisfies Partial<InstanceType<typeof DefaultOverrides>>;`

Then test by starting a run and seeing that the icon for Yungoos no longer has the floating pixel.

## Checklist
